src/CMakeLists.txt

```diff
@@ -253,17 +253,26 @@ ENDFOREACH(FILE)
 FILE(WRITE ${CMAKE_BINARY_DIR}/file_list.txt "${WRITE_FILE}")
 
 ######################################################################################
-# Architecutre to build forte on
+# Architecutre to build 4diac FORTE on
 ######################################################################################
 set(FORTE_ARCHITECTURE "None" CACHE STRING "Architecture to build FORTE on")
 GET_PROPERTY(architectures GLOBAL PROPERTY FORTE_ARCHITECTURES)
 list(SORT architectures)
 set_property(CACHE FORTE_ARCHITECTURE PROPERTY STRINGS None ${architectures})
 
+######################################################################################
+# Device to be used in this 4diac FORTE instance
+######################################################################################
+set(FORTE_DEVICE "RMT_DEV" CACHE STRING "Device to be used in this 4diac FORTE instance")
+GET_PROPERTY(devices GLOBAL PROPERTY FORTE_DEVICES)
+list(SORT devices)
+set_property(CACHE FORTE_DEVICE PROPERTY STRINGS None ${devices})
+
```

Comment:
I'm not sure if there should be some validation for the possible values of FORTE_DEVICE - the set_property call only limits the values that can be selected via CMakeGUI

```
if(NOT ${FORTE_DEVICE} IN_LIST devices)
  message(FATAL_ERROR "Invalid device selected: ${FORTE_DEVICE}")
endif()
```

```
if(NOT ${FORTE_DEVICE} IN_LIST devices)
  set(FORTE_DEVICE "RMT_DEV" CACHE STRING "Device to be used in this 4diac FORTE instance")
endif()
```

Reply:
Goo point. However sofar we also do not have it for architecture and there it also works fine. In the worst case you would get a compile error.